packages/pcd/gpc-pcd/test/gpc-pcd.spec.ts

```diff
@@ -66,15 +72,26 @@ describe("GPCPCD should work", async function () {
               isMemberOf: "admissibleOwners"
             }
           }
+        },
+        ticketPOD: {
+          entries: {
+            ticketID: {
+              isRevealed: false,
+              isMemberOf: "admissibleTickets"
+            }
+          }
         }
       },
       tuples: {
         pair: { entries: ["pod0.A", "pod0.E"], isMemberOf: "admissiblePairs" }
       }
     };
 
-    const pod = POD.sign(sampleEntries, privateKey);
-    const podPCD = new PODPCD(uuid(), pod);
+    const pod0 = POD.sign(sampleEntries0, privateKey);
```

Comment:
I'd suggest switching to a name other than pod0 everywhere, to ensure the system is no longer relying on that hardcoded name.